Packages/ANCIENT-MEGALITHIC-MYSTERY-PACKAGE/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="atg-location" content="Cusco">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" type="text/css" href="../../css/main.css">
+	<link rel="stylesheet" type="text/css" href="../../css/global.css">
+	<!-- PACKAGE STYLES -->
+	<link rel="stylesheet" type="text/css" href="../../css/packages.css">
+	<title>ANCIENT MEGALITHIC MYSTERY PACKAGE</title>
```

Comment:
Don't use all caps please - just capitalize first letters -   i can't help it but i hate the word "package" - it's so status quo and reminds me of the "old way" of doing things - i'd rather disrupt and call them "packs"

Reply:
ok ill change it to packs